frontend/src/validation-schemas/helpers/validation/empty-file-list-validation.helper.ts

```diff
@@ -0,0 +1,14 @@
+import { Joi } from 'helpers/helpers';
+import { getFileFromFileList } from 'helpers/helpers';
+
+const emptyFileListValidation = (image: FileList, helpers: Joi.CustomHelpers<FileList>): FileList | Joi.ErrorReport => {
+  const file = getFileFromFileList(image);
+
+  if (!file) {
+    return image;
+  } else {
+    return helpers.error('file.fileListIsNotEmpty');
+  }
```

Comment:
We can omit the `else` statement, because we use `return`

```suggestion
    if (!file) {
    return image;
  }
    return helpers.error('file.fileListIsNotEmpty');
```